Add alt text for svg_stroke images (#22852)
mdn/content: files/en-us/web/svg/tutorial/fills_and_strokes/index.md

In addition to its color properties, there are a few other attributes available to control the way a stroke is drawn on a line.

> ![The stroke-linecap attribute changes the look of these stroke's ends](svg_stroke_linecap_example.png)

Reviewer: ```suggestion
![The stroke-linecap attribute changes the look of these stroke's ends: square adds a square cap, round provides a rounded cap, and butt removes capping](svg_stroke_linecap_example.png)
```

Use `stroke-linejoin` to control how the joint between two line segments is drawn.

> ![The stroke-linejoin attribute changes the look at the point where two lines join.](svg_stroke_linejoin_example.png)

Reviewer: ```suggestion
![The stroke-linejoin attribute changes the look at the point where two lines join, with miter created an angled join, round rounding the corner, and bevel creating a beveled edge, flattening the corner .](svg_stroke_linejoin_example.png)
```

Finally, you can also use dashed line types on a stroke by specifying the `stroke-dasharray` attribute.

> ![Shows two custom lines using different stroke-dasharray attributes.](svg_stroke_dasharray_example.png)

Reviewer: ```suggestion
![Two custom dashed lines, one with evenly spaced dashes and the other using a long-dash short dash using a stroke-dasharray attribute value.](svg_stroke_dasharray_example.png)
```
